orders-and-customers/orders/order-items.md

```diff
@@ -212,6 +228,18 @@ The Bearer token to grant access to the API
               "currency": "GBP",
               "formatted": "£0"
             }
+          },
+          "tax": {
+            "unit": {
```

Comment:
What is unit/value?

Reply:
`Unit` is for the tax amount per item where as `value` is the total for that item based on what quantity you have. I'll see if there's a good place for this to be added in as I don't think it's explained anywhere for the other fields.

Reply:
Why not `unit` and `total`?

Reply:
This is what was initially stipulated way back when. It's a breaking change if we alter this.